src/pybind/mgr/cephadm/module.py

```diff
@@ -208,7 +208,7 @@ class CephadmOrchestrator(orchestrator.Orchestrator, MgrModule,
         ),
         Option(
             'container_image_haproxy',
-            default='haproxy',
+            default='docker.io/library/haproxy:2.3',
```

Comment:
```suggestion
            default=DEFAULT_HAPROXY_IMAGE,
```

(nit) make similar to how the other image defaults are defined